docs/registration.md

```diff
@@ -383,21 +383,31 @@ All classes and functions documented here are directly importable from `textx` m
   parameters
 - `clear_language_registrations()` - deletes all languages registered
   programmatically. Note: languages registered through `setup.py` won't be removed
-- `metamodel_for_language(language_name)` - returns a meta-model for the given language name
+- `metamodel_for_language(language_name, **kwargs)` - returns a meta-model for
+  the given language name. `kwargs` are additional keyword arguments passed to
+  meta-model factory callable, similarly to `metamodel_from_str/file`.
```

Comment:
Good idea... but I am not sure about the consequences / use case: I just wondered if we a language may have some restrictions, some decisions about meta model already made by the language designer. 

--> Is it a use case that I request a metamodel for a (predefined, ready-to-use) language with other meta model parameters that the language designer defined for me...?

Reply:
Good point. I found this use case when trying to built meta-model with a particular set of user classes and figured out that a need to override other stuff may be needed. Let's say that those kwargs should be whatever language designer is defined as overridable. Those params are passed to a meta-model callable factory so the factory can decide what to do with those.

Probably some more clarification needed in the docs.

Reply:
This is ok. Since a meta model provider (language designer) can prevent such actions and throw and error.

For me it is ok.

Reply:
Yes, you are right. The language designer can decide what to do with passed parameters.